__tests__/news.test.js

```diff
@@ -178,4 +208,31 @@ describe("GET /api/articles/:article_id/comments", () => {
                 expect(response.body.selectedComments.every((comment) => comment.article_id === 5)).toBe(true);
             });
     });
+
+    it("Should return an empty array of comments for the non-existing article_id", () => {
+        return request(app)
+            .get("/api/articles/2/comments")
+            .expect(200)
+            .then((response) => {
+                expect(response.body.selectedComments).toEqual([]);
+            });
+    });
```

Comment:
Isn't this looking at article 2?  This does exist, so your test description isn't completely correct